client/ayon_core/hosts/hiero/api/startup/Python/StartupUI/otioimporter/OTIOImport.py

```diff
@@ -153,18 +146,13 @@ def prep_url(url_in):
     if url.startswith("file://localhost/"):
         return url.replace("file://localhost/", "")
 
-    url = '{url}'.format(
-        sep=url.startswith(os.sep) and "" or os.sep,
-        url=url.startswith(os.sep) and url[1:] or url
-    )
+    url = "{url}".format(url=url.startswith(os.sep) and url[1:] or url)
```

Comment:
Uh - what? There's something wrong with this logic.
This will make `url = "True"` if url startswith with `os.sep`

I think this was intended to be: 

```suggestion
    url = url[1:] if url.startswith(os.sep) else url
```

Or:

```suggestion
    if url.startswith(os.sep):
        url = url[1:]
```

Or:

```suggestion
    url = url.lstrip(os.sep)
```
_this last one may strip more however_